mbed_app.json

```diff
@@ -209,6 +209,38 @@
             "drivers.uart-serial-txbuf-size"            : 256,
             "drivers.uart-serial-rxbuf-size"            : 1024,
             "mbed-cloud-client-update-buffer-size"      : 1024
+        },
+        "EP_AGORA": {
+            "target.bootloader_img"                     : "tools/mbed-bootloader-ep_agora-external_kvstore-spif-v4.1.2.bin",
+            "target.header_offset"                      : "0x10000",
+            "target.app_offset"                         : "0x10400",
+            "target.components_remove"                  : ["QSPIF"],
+            "target.components_add"                     : ["SPIF"],
+            "update-client.bootloader-details"          : "0x7394",
+            "update-client.application-details"         : "(MBED_CONF_STORAGE_FILESYSTEM_INTERNAL_BASE_ADDRESS + MBED_CONF_STORAGE_FILESYSTEM_RBP_INTERNAL_SIZE)",
+            "update-client.storage-address"             : "(MBED_CONF_STORAGE_FILESYSTEM_EXTERNAL_BASE_ADDRESS + MBED_CONF_STORAGE_FILESYSTEM_EXTERNAL_SIZE)",
+            "update-client.storage-size"                : "((MBED_ROM_START + MBED_ROM_SIZE - APPLICATION_ADDR) * MBED_CONF_UPDATE_CLIENT_STORAGE_LOCATIONS)",
+            "update-client.firmware-header-version"     : "2",
+            "mbed-cloud-client.update-storage"          : "ARM_UCP_FLASHIAP_BLOCKDEVICE",
+            "storage_filesystem.internal_base_address"  : "(MBED_ROM_START + MBED_BOOTLOADER_SIZE)",
+            "storage_filesystem.rbp_internal_size"      : "(32*1024)",
+            "storage.storage_type"                      : "FILESYSTEM",
+            "storage_filesystem.filesystem"             : "LITTLE",
+            "storage_filesystem.blockdevice"            : "SPIF",
+            "storage_filesystem.external_base_address"  : "(0x0)",
+            "storage_filesystem.external_size"          : "(1024*1024*1)",
+            "drivers.uart-serial-rxbuf-size"            : 1024,
+            "drivers.uart-serial-txbuf-size"            : 1024,
+            "lwip.ipv4-enabled"                         : true,
+            "lwip.ipv6-enabled"                         : true,
+            "lwip.ppp-enabled"                          : true,
+            "lwip.tcp-enabled"                          : true,
+            "lwip.ethernet-enabled"                     : false,
+            "lwip.mem-size"                             : 22000,
+            "nsapi.dns-response-wait-time"              : 30000,
+            "nsapi.default-cellular-apn"                : "\"phone\"",
```

Comment:
I'd remove this line since it's up to the user to specify the APN for their SIM card. It'd probably be good to put some dummy cell credentials in the `*` target override section above like was done [here](https://github.com/ARMmbed/mbed-os-example-pelion/blob/master/mbed_app.json#L24) for the `mbed-os-example-pelion` application.